### DISCUSSION TITLE
Video frame flicks, after some seek operations. AndroidTV

### DISCUSSION THREAD

**Discussion by Dav2015** [TheWidlarzGroup/react-native-video#3559] [Q&A]
### What are you trying to do?

Playing an mp4 video on AndroidTV using react-native-video v6 beta. After some seek backward and forward operations, the video starts to flick. It looks like a buffering issue. The same issue also occurred on the v5 version.

### What have you tried?

Tried different `bufferConfig` values but did not find a solution.

A screen recording of the issue was attached:

https://github.com/react-native-video/react-native-video/assets/15325275/33aaa92b-00bc-4b1a-8e7a-c1e49ce169dc

### Relevant code

```tsx
bufferConfig={{
                minBufferMs: 3000, // 3 seconds
                maxBufferMs: 7000, // 7 seconds
                bufferForPlaybackMs: 3000, // 3 seconds
                bufferForPlaybackAfterRebufferMs: 3000, // 3 seconds
                maxHeapAllocationPercent: 0.5, // 50%
                minBackBufferMemoryReservePercent: 0.05, // 5%
                minBufferMemoryReservePercent: 0.05, // 5%
            }}
```

### Version

v6

### Platforms

Android TV

**Comment by TWG-AI** [comment]
This sounds like a visual glitch after seeking on Android TV — the video flickering after back-and-forth seeks is often a buffering artifact. To help investigate, could you share:

1. Your exact `react-native-video` version
2. Android TV device/emulator and OS version
3. A screen recording of the issue
4. Your current `bufferConfig` (or confirm you've tried defaults)

A minimal reproducible example would help the team narrow down whether this is in the ExoPlayer surface rendering or the seek pipeline.

**Comment by moskalakamil** [comment]
hey @Dav2015, please create an issue with this instead of disscussion
